.github/ci-test-connections.yaml

```diff
@@ -35,3 +35,19 @@ connections:
     schema:
     login:
     password:
+  - conn_id: gcp_conn
+    conn_type: google_cloud_platform
+    description: null
+    extra: null
+  - conn_id: google_cloud_default
+    conn_type: google_cloud_platform
+    description: null
+    extra: null
```

Comment:
Why are we adding this connection as opposed to using the default Airflow one which has the same name?